src/main/java/bisq/core/locale/CurrencyUtil.java

```diff
@@ -95,13 +87,17 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
     public static List<CryptoCurrency> createAllSortedCryptoCurrenciesList() {
         final List<CryptoCurrency> result = new ArrayList<>();
 
+        final ServiceLoader<SpecificAltCoinAddressValidator> loader = ServiceLoader.load(SpecificAltCoinAddressValidator.class);
+        for (SpecificAltCoinAddressValidator validator : loader) {
+            result.add(new CryptoCurrency(validator.getCurrencyCode(), validator.getCurrencyName(), validator.isAsset()));
```

Comment:
My tase tells me that validator should not hold general info about currency, but if we'd like to introduce an additional class (i.e. BchRegistrant) with altcoin description that would force users to create an additional class. What's your take on this?